nestkernel/event.h

```diff
@@ -309,6 +309,8 @@ class Event
    */
   Time stamp_;
 
+  unsigned long steps_;
```

Comment:
This is actually "doppeltgemoppelt": If we cache the steps value already in the `Time` object `stamp_`, we do not need to store it additionally in the `Event` object.